aiomysql/__init__.py

```diff
@@ -23,6 +23,11 @@
 
 """
 
+try:
+    import pkg_resources
```

Comment:
The use of `pkg_resources` is highly discouraged. Replace it with `importlib_metadata` (add an optional backport to the runtime deps if needed — expecting an import error is not necessary).

Reply:
with `setuptools-scm`'s `write_to` this won't be needed